build/Dockerfile.ci

```diff
@@ -42,6 +42,12 @@ WORKDIR /build/windows-machine-config-operator/hcsshim/
 COPY hcsshim/ .
 RUN GOOS=windows go build ./cmd/containerd-shim-runhcs-v1
 
+# Build containerd
```

Comment:
There is a `# Build containerd` stanza a few lines lines above that should be modified instead of adding this one

Reply:
Thanks for the catch, looks like this came from my local testing